app/controllers/api/base_controller.rb

```diff
@@ -39,8 +39,8 @@ class BaseController < ActionController::Base
 
     rescue_from Foreman::MaintenanceException, :with => :service_unavailable
 
-    def get_resource
-      instance_variable_get(:"@#{resource_name}") || raise('no resource loaded')
+    def get_resource(options = {})
```

Comment:
I think 2.0 has limited support, all keyword arguments would have to be optional. Maybe that's enough for this case though.